packages/react-inbox/src/components/Message/index.tsx

```diff
@@ -58,44 +60,44 @@ const Message: React.FunctionComponent<MessageProps> = ({
     });
   }, [created]);
 
-  const actions = useMemo(() => {
-    return [{ href: data?.clickAction, label: "View Details" }];
-  }, [data]);
+  const actions = useMemo(() => [{
+    href: data?.clickAction,
+    label: "View Details",
+  }], [data]);
 
   const options = useMemo(
     () =>
       [
         !read &&
           trackingIds?.readTrackingId && {
-            label: "Mark as Read",
-            onClick: () => {
-              if (!trackingIds?.readTrackingId) {
-                return;
-              }
+          label: "Mark as Read",
+          onClick: () => {
+            if (!trackingIds?.readTrackingId) {
+              return;
+            }
 
-              markMessageRead(messageId, trackingIds?.readTrackingId);
-            },
+            markMessageRead(messageId, trackingIds?.readTrackingId);
           },
+        },
 
         read &&
           trackingIds?.unreadTrackingId && {
-            label: "Mark as Unread",
-            onClick: () => {
-              if (!trackingIds?.readTrackingId) {
-                return;
-              }
+          label: "Mark as Unread",
+          onClick: () => {
+            if (!trackingIds?.readTrackingId) {
+              return;
+            }
 
-              markMessageUnread(messageId, trackingIds?.unreadTrackingId);
-            },
+            markMessageUnread(messageId, trackingIds?.unreadTrackingId);
           },
+        },
         /*{
         label: "Delete",
         onClick: () => {},
       },*/
       ].filter(Boolean),
-    [read, messageId, trackingIds]
+    [read, trackingIds?.readTrackingId, trackingIds?.unreadTrackingId, markMessageRead, messageId, markMessageUnread],
```

Comment:
we need to add `markMessageUnread` and `markMessageRead` and each tracking id separatly here?